web/themes/joinup/templates/form/input--checkbox.html.twig

```diff
@@ -10,4 +10,4 @@
  * @see template_preprocess_input()
  */
 #}
-<input{{ attributes.addClass('mdl-checkbox__input') }} />{{ children }}
+<input{{ attributes.addClass('mdl-checkbox__input') }} />
```

Comment:
The children variable is empty for checkbox elements, so I think that it can be removed.